docs/ISSUE_TRIAGE.md

```diff
@@ -0,0 +1,288 @@
+1. Assign a reviewer based on who was handling the linked issue or what code or feature the PR touches (look at who was the last to make changes there if all else fails).
+
+## GitHub issue management workflow
+
+The following section describes the triage workflow for new GitGHub issues that get created.
+
+### GitHub Issue: Bug
+
+This workflow starts off with a GitHub issue of type bug being created.
+
+1. Collaborator or maintainer creates a GitHub bug using the appropriate GitHub issue template
+2. By default a bug will be created with the `type/bug` and `needs-triage` labels
+
+The following flowchart defines the workflow,
+
+
+```                                                                                                                                                     
+                                               +--------------------------+                                                                              
+                                               | New bug issue opened/more|                                                                              
+                                               | information added        |                                                                              
+                                               +-------------|------------+                                                                              
+                                                             |                                                                                           
+                                                             |                                                                                           
+   +----------------------------------+  NO   +--------------|-------------+                                                                             
+   | label: triage/needs-information  ---------  All required information  |                                                                             
+   |                                  |       |  contained in issue?       |                                                                             
+   +-----------------------------|----+       +--------------|-------------+                                                                             
+                                 |                           | YES                                                                                       
+                                 |                           |                                                                                           
+   +--------------------------+  |                +---------------------+ YES +---------------------------------------+                                  
+   |label:                    |  |                |  Dupicate Issue?    ------- Comment `Duplicate of #<issue number>`                                   
```

Comment:
typo/spelling: Change "Dupicate" to "Duplicate".

Reply:
addressed